File: Java/Kriti.java
public class Kriti{
	public static void main(String[] args) {
		java.util.Scanner in = new java.util.Scanner(System.in);

		int numString = in.nextInt();
		
		String[] arr = new String[numString];
		
		for (int i = 0; i < arr.length; i++) {
			arr[i] = in.next();
		}

		int numQuery = in.nextInt();
		
		int[] count = new int[numQuery];
		
		int l = 0, r = 0;
		String s = "";

		for (int x = 0; x < numQuery; x++) {
			l = Integer.parseInt(in.next());
			r = Integer.parseInt(in.next());
			s = in.next();

			int counter = 0;

			for (int y = (l - 1); y < r; y++){
				if (s.equals(arr[y])) counter++;
			}
			count[x] = counter;
		}

		for (int x = 0; x < count.length; x++) {
			System.out.println(count[x]);
		}
	}
}

File: C++/Kriti.cpp
#include <iostream>
#include <string>

int main(){
	int numString, numQuery;

	std::cin >> numString;
	
	std::string arr[numString];

	for (int i = 0; i < numString; i++){
		std::cin >> arr[i];
	}

	std::cin >> numQuery;
	
	int count[numQuery];

	for (int i = 0; i < numQuery; i++){
		int l, r;
		std::string s;
		std::cin >> l >> r >> s;

		int counter = 0;
		
		for (int j = (l - 1); j < r; j++){
			counter += (s == arr[j]) ? 1 : 0;
		}
		count[i] = counter;
	}

	for (int i = 0; i < numQuery; i++){
		std::cout << count[i] << std::endl;
	}
}